apple/internal/xcarchive.bzl

```diff
@@ -33,18 +36,25 @@ def _xcarchive_impl(ctx):
     arguments.add("--bundle", bundle_info.archive.path)
     arguments.add("--output", xcarchive.path)
 
-    for dsym in dsym_info.direct_dsyms:
-        arguments.add("--dsym", dsym.path)
+    arguments.add_all(
+        dsym_info.transitive_dsyms,
+        map_each = _to_path,
```

Comment:
Is this needed? Just a little surprised it is.

Reply:
Turns out it's not, good catch